packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -60,6 +60,15 @@ sh.set('-e')
 // will ensure those escaped double quotes are still escaped after processing the template.
 Handlebars.registerHelper('script', (object) => object.replaceAll('"', '\\"'))
 
+Handlebars.registerHelper('pascalcase', function (str) {
+    const result = str.replace(/(^\w|-\w)/g, function (txt) {
+        return txt.replace(/-/, '').toUpperCase()
+    })
+    return str.replace(/(^\w|-\w)/g, function (txt) {
+        return txt.replace(/-/, '').toUpperCase()
+    })
+})
```

Comment:
I think we are missing to handle special symbols like @ and / underscores, dots, or other special symbols that are valid on NPM packages.